cmd/encryption-v1.go

```diff
@@ -77,7 +77,7 @@ const (
 // hasServerSideEncryptionHeader returns true if the given HTTP header
 // contains server-side-encryption.
 func hasServerSideEncryptionHeader(header http.Header) bool {
-	return crypto.S3.IsRequested(header) || crypto.SSEC.IsRequested(header)
+	return crypto.S3.IsRequested(header) || crypto.SSEC.IsRequested(header) || crypto.S3KMS.IsRequested(header)
```

Comment:
Why is `crypto.S3KMS.IsRequested(header)` being added? What will happen if a client sets a SSE-KMS encryption headers and makes a putobject call? I see it is being used in `PostPolicyBucketHandler`, `PutObjectHandler`, `NewMultipartUploadHandler`. Will this affect any of these API behavior?

Reply:
This is needed for AWS S3 compatibility, notice it is only for PutObjectPart and CopyObjectPart. 

But there are other major issues with the implementation https://github.com/minio/minio/issues/6579 one such is this.

Reply:
This not necessary - We don't support SSE-KMS at the moment and the server will return `Not Implemented`